hca/upload/api_client.py

```diff
@@ -32,6 +32,52 @@ def credentials(self, area_uuid):
                     content=response.content))
         return response.json()
 
+    def checksum_status(self, area_uuid, filename):
+        url = "{api_url_base}/area/{uuid}/{filename}/checksum".format(api_url_base=self.api_url_base, uuid=area_uuid,
+                                                                      filename=filename)
+        response = requests.get(url)
+        if not response.ok:
+            raise RuntimeError(
+                "GET {url} returned {status}, {content}".format(
+                    url=url,
+                    status=response.status_code,
+                    content=response.content))
+        return response.json()
```

Comment:
The methods in this file have become very repetitive.  Time to dry them up.  It seems to me you could factor out a _get() and _post() method and delete half this file.

Reply:
CodeClimate can be a bit overzealous :)  Eliminating **every** *similar* piece of code would go way past the point of diminishing returns.  Just consider its feedback suggestions of where to look.